src/l10n.json

```diff
@@ -269,6 +278,19 @@
     "report.promptPersonal": "Please say where the personal contact information is shared (For example: Notes & Credits, sprite name, project text, etc.)",
     "report.promptGuidelines": "Please be specific about why this project does not follow our Community Guidelines",
     "report.promptImage": "Please say the name of the sprite or the backdrop with the inappropriate image",
+    "report.promptDontLikeIt": "Scratch projects are made by people of all ages and levels of experience. If you don't like this project because you feel it can be improved upon, we encourage you to share constructive feedback directly with the creator.",
+    "report.promptTips": "Here are tips on sharing constructive feedback:",
+    "report.tipsSupportive": "Be supportive and encouraging.",
+    "report.tipsConstructive": "Leave a comment telling them what you like, but also what they could do to make the project better.",
+    "report.tipsSpecific": "Try to be specific with your feedback. For instance: The controls to move the character did not work.",
+    "report.promptDoesntWork": "A Scratch project, like any other application, may contain a few bugs. That is expected and completely okay!",
+    "report.promptDoesntWorkTips": "We encourage you to share any issues you discover directly with the creator of the project. It's also helpful to provide suggestions on how they may improve their project, if possible.",
+    "report.promptTooHard": "If you feel a project could be easier, we encourage you to share that feedback directly with the creator of the project. Or remix it yourself and make it as easy or hard as you like!",
+    "report.promptMisleading": "Tell us more about how it's tricking or misleading people",
+    "report.promptFaceReveal": "Scratch allows people to use pictures of their face in creative projects like games, stories, or animations. However, projects that are just a picture of their face, or which focus entirely on their physical appearance, aren't allowed on Scratch. If this is a 'face reveal', or the project focuses on the person's appearance, please click report.",
```

Comment:
The button says "Send", not "report" (in English, at least). Is there a way in `react-intl` to reference messages inside other messeages so that `report.send` can be substituted in?

Reply:
Good point! I made a change to substitute in `report.send`.